tracim/fixtures/content.py

```diff
@@ -37,12 +39,25 @@ def insert(self):
         role_api = RoleApi(
             current_user=admin,
             session=self._session,
+            config=self._config,
         )
 
         # Workspaces
-        w1 = admin_workspace_api.create_workspace('w1', save_now=True)
-        w2 = bob_workspace_api.create_workspace('w2', save_now=True)
-        w3 = admin_workspace_api.create_workspace('w3', save_now=True)
+        w1 = admin_workspace_api.create_workspace(
+            'w1',
```

Comment:
Rework of Content fixture naming is done here : https://github.com/tracim/tracim_backend/pull/83 : https://github.com/tracim/tracim_backend/pull/83/commits/03342866aa71ed7f69004f5078782cc519050d6b ( because i needed to add new files for tests)